src/basilisp/lang/obj.py

```diff
@@ -41,7 +43,7 @@ def _dec_print_level(lvl: PrintCountSetting) -> PrintCountSetting:
     return lvl
 
 
-def _process_kwargs(**kwargs: Unpack[PrintSettings]) -> PrintSettings:
+def process_kwargs(**kwargs: Unpack[PrintSettings]) -> PrintSettings:
```

Comment:
```suggestion
def process_lrepr_kwargs(**kwargs: Unpack[PrintSettings]) -> PrintSettings:
```

Since this is exported, the meaning of the name is a bit more important. I hope this will be a little more explicit.

Reply:
I've reverted this change to the original _process_kwargs name, because with the new split of the map_lrepr into two, there is no need to expose this fn outside of obj.py